packages/dd-trace/src/appsec/addresses.js

```diff
@@ -20,5 +20,7 @@ module.exports = {
   HTTP_CLIENT_IP: 'http.client_ip',
 
   USER_ID: 'usr.id',
-  WAF_CONTEXT_PROCESSOR: 'waf.context.processor'
+  WAF_CONTEXT_PROCESSOR: 'waf.context.processor',
+
+  RASP_IO_URL: 'server.io.net.url'
```

Comment:
I'm not convinced by the address identifier. I think we should not tie it to RASP, but to SERVER or maybe the vulnerability SSRF. WDYT?